base/essentials.jl

```diff
@@ -151,9 +151,29 @@ macro isdefined(s::Symbol)
     return Expr(:escape, Expr(:isdefined, s))
 end
 
+# can be used in place of `@pure` (supposed to be used for bootstrapping)
 macro _pure_meta()
```

Comment:
can we delete `_pure_meta` now?

Reply:
(and `@pure`?)

Reply:
It's technically internal so we should be able to do so, but I suspect there's a decent amount of package code that uses it, so we should probably at least go through some kind of deprecation process. Or maybe just make PRs to affected packages to not use it if `VERSION` ≥ v"1.9".

Reply:
is anyone using it correctly? I don't think @vtjnash has added uses to packages, and as far as I understand, he is the only person allowed to use it.

Reply:
https://juliahub.com/ui/Search?q=pure&type=symbols&t=macro

Reply:
Now all of these internal macros are no-op when used outside of Base ;)